design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,354 @@
+# Ignore changes
+
+* Owners: Lovro Sviben (@lsviben)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Background
+
+When Crossplane manages a resource, it reconciles all the parameters
+under  `spec.forProvider` to the external providers both during the
+creation and subsequent updates to the managed resource. But in some cases,
+a parameter of the external resource could change due to it being managed
+outside Crossplane. In the current state, if that happens, Crossplane
+will try to "fix" the situation, and update the parameter back to the one it
+has in the `spec.forProvider` as it is designed to be the only source of truth.
+
+Some examples of this case:
+- [AutoScalingGroup DesireCapacity], where when using an autoscaler, the
+  parameter gets changed externally.
+- [DynamoDB Tables] with using autoscaling ([Policy] + [Target]). The
+  autoscaler scales the read and write capacity, and then Crossplane resets
+  it back to the `readCapacity` and `writeCapacity` defined on the Table,
+  which effectively makes autoscaling useless.
+- [NodeGroup] while using a node autoscaler. There is a race condition
+  between crossplane and node autoscaler. Node autoscaler increases the
+  `desiredSize` due to resource requirement but crossplane decrease it to
+  `desiredSize` set in composition.
+- GCP [Node Pool], where when `management.autoUpgrade` is enabled. After
+  creation, the provider late initializes `spec.forProvider.version`,
+  which when the nodes get upgraded starts conflicting with the 
+  external nodes version. This causes an endless loop of downgrades
+  and upgrades.
+- Tags are a use case that affects multiple resources. Many cloud users
+  have external systems that add tags to resources for billing, tracking
+  or similar purposes. This can conflict with the tags set in 
+  `spec.forProvider.tags`.
```

Comment:
This use case seems a little different from the others. Tags are a set, so I could imagine folks wanting to configure something like "ensure these tags are always present, but also allow other tags to be added". This is a little different from "set these tags initially, but after that I don't care what the tags are". In the former case we'd always enforce the tags you specified, but not delete tags added by other entities. In the latter case, we'd only set the specified tags at creation time. If some other entity deleted them, we would not correct that.

I don't think this observation invalidates the design being put forward here - at the very least it's probably a "good enough" solution for the tags use case.